bookkeeper-server/src/main/java/org/apache/bookkeeper/client/LedgerFragmentReplicator.java

```diff
@@ -383,6 +403,9 @@ public void readComplete(int rc, LedgerHandle lh,
                         .computeDigestAndPackageForSending(entryId,
```

Comment:
Because the read and write operation at the same place, should we just use a `replicateThrottle` to control them is enough?

Reply:
@zymap  PTAL again